tools/mo/openvino/tools/mo/analysis/json_print_new_frontend.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2022 Intel Corporation
```

Comment:
I recommend to move this into mo/moc_frontend/analysis.py because this new analysis pass is applicable only for new frontends. For legacy case we still need to use old one.

Reply:
done